selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestWorkspaceServiceClient.java

```diff
@@ -203,26 +212,33 @@ public String getServerAddressByPort(String workspaceId, int port) throws Except
         .getRuntime()
```

Comment:
This method looks suspicious in SPI branch. It is probable that we should remove usage of it. Can you deprecate it to show that it should not be used in future?

Reply:
ok